docs/community.rst

```diff
@@ -0,0 +1,95 @@
+Community
+---------
+
+You can check out Kagi on GitHub at https://github.com/justinmayer/kagi/.
+
+.. _communication_channels:
+
+Communication channels
+----------------------
+
+* Don't hesitate to `create issues <https://github.com/justinmayer/kagi/issues>`
+  on the github repository.
+
+.. _how-to-contribute:
+
+How to contribute
+-----------------
+
+Thanks for your interest in contributing to *Kagi*!
+
+.. note::
+
+    We love community feedback and are glad to review contributions of any
+    size - from typos in the documentation to critical bug fixes - so don't be
+    shy!
+
+Report bugs
+:::::::::::
+
+Report bugs at https://github.com/Kinto/kinto/issues/new
+
+If you are reporting a bug, please include:
+
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix bugs
+::::::::
+
+Check out the `open bugs <https://github.com/justinmayer/kagi/issues>`_ - anything
+tagged with the |good-first-issue label|_ could be a good choice for newcomers.
+
+.. |good-first-issue label| replace:: **good-first-issue** label
+.. _`good-first-issue label`: https://github.com/justinmayer/kagi/labels/good%20first%20issue
+
+
+Implement features
+::::::::::::::::::
+
+Look through the GitHub issues for features. Anything tagged with |enhancement|_
+is open to whoever wants to implement it.
+
+.. |enhancement| replace:: **enhancement**
+.. _enhancement:  https://github.com/justinmayer/kagi/labels/enhancement
+
+Write documentation
+:::::::::::::::::::
+
+*Kagi* could always use more documentation, whether as part of the
+official docs, in docstrings, or even on the Web in blog posts,
+articles, and such.
+
+This official documentation is maintained in `GitHub <https://github.com/justinmayer/kagi/>`_.
+The ``docs`` folder contains the documentation sources in `reStructuredText <https://en.wikipedia.org/wiki/ReStructuredText>`_ format. And you can generate the docs locally with::
+
+    make docs
```

Comment:
Since it seems we're going to remove the Makefile, we'll probably want to change this (and any other `make` references) to the corresponding Invoke… invocations. :wink: